docs/user/convert/function_usage.rst

```diff
@@ -879,6 +890,16 @@ delimiters for a map:
 
     { name = "numbers", transform = "parseMap('int -> string', $2, '->', ',')" }
 
+transformListItems
```

Comment:
Can you add the ticket(s) for rounding LISP support in the converters? </rimshot>